src/node_report.cc

```diff
@@ -782,6 +792,38 @@ static void PrintRelease(JSONWriter* writer) {
   writer->json_objectend();
 }
 
+static void PrintRequestInfo(JSONWriter* writer,
+                             Environment* env,
+                             const char* trigger) {
+  bool printOwnerInfo = strcmp(trigger, "OOMError") != 0;
```

Comment:
```suggestion
  bool print_owner_info = strcmp(trigger, "OOMError") != 0;
```

https://github.com/nodejs/node/blob/50a413183e92c690e1d9c594d59603cc8e717d62/doc/contributing/cpp-style-guide.md#snake_case-for-local-variables-and-parameters